tasks/configure_user.yaml

```diff
@@ -0,0 +1,41 @@
+---
+- name: "check for required fact - name"
+  fail:
+    msg: "missing required fact: name"
+  with_items: "{{ users }}"
+  when: ( user.name is not defined )
```

Comment:
It should also check if users is defined at first place.

Reply:
+1